environments/host_vars/website.yml

```diff
@@ -0,0 +1,7 @@
+---
+ansible_ssh_host: 165.227.4.15
+ansible_ssh_port: 22
+ansible_ssh_user: root
+email: pythondev.slack@gmail.com
+systemd_enabled: true
+travis_ci: false
```

Comment:
This is defined in the host definition, not the role. But that said - we could default them to sane values in roles and only override for test. In fact I would think we should make the travis_ci one more generic (in_ci or something, since travis is one of many)